ehr2vec/notebooks/scripts/time2event_vdl_sim.py

```
@@ -0,0 +1,325 @@
+        k (int): Second time index running to T_observed.
+        A (int): Which curve to estimate, 0 or 1.
+    """
+    if patient['A'] != A:
+        return 0
+    
+    if k>t:
+        return 0
+    if patient['A'] == 1:
+        g = patient['propensity']
+    else:
+        g = 1 - patient['propensity']
```

Ruff

Use ternary operator `g = patient['propensity'] if patient['A'] == 1 else 1 - patient['propensity']` instead of `if`-`else`-block

Replace `if`-`else`-block with `g = patient['propensity'] if patient['A'] == 1 else 1 - patient['propensity']`

(SIM108)